github/resource_github_branch_protection.go

```diff
@@ -96,6 +96,21 @@ func resourceGithubBranchProtection() *schema.Resource {
 							Type:     schema.TypeBool,
 							Optional: true,
 						},
+						"required_approving_review_count": {
+							Type:        schema.TypeInt,
+							Optional:    true,
+							Description: "Number of approvals required to merge a pull request",
+							Default:     1,
+							// Old version of terraform, could not utilize validation.IntBetween
+							ValidateFunc: func(i interface{}, k string) (s []string, es []error) {
```

Comment:
You can use:
```go
import "github.com/hashicorp/terraform/helper/validation"
...

ValidateFunc: validation.IntBetween(1, 6),
```

Reply:
Will be needed to add this on `vendor.json`
```json
		{
			"checksumSHA1": "nEC56vB6M60BJtGPe+N9rziHqLg=",
			"path": "github.com/hashicorp/terraform/helper/validation",
			"revision": "41e50bd32a8825a84535e353c3674af8ce799161",
			"version": "v0.11.7",
			"versionExact": "v0.11.7"
		},
```

Reply:
Ah, you are right, my comment was indeed incorrect.  Thanks for the comment!
90d250897ec31ecbf9f5a312fd70749ae6c3836b